lib/wovnrb/api_translator.rb

```diff
@@ -23,11 +23,15 @@ def translate(body)
 
       case response
       when Net::HTTPSuccess
-        if response.header['Content-Encoding'] == 'gzip'
-          response_body = Zlib::GzipReader.new(StringIO.new(response.body)).read
-
+        begin
+          response_body =
+            if response.header['Content-Encoding'] == 'gzip'
+              Zlib::GzipReader.new(StringIO.new(response.body)).read
+            else
+              response.body
```

Comment:
We expect the API to send gzip-encoded data, if data is not gzip-encoded we cannot trust its validity nor its format, so why did you change into returning `response.body`?

Reply:
I added this to make it work in my local env. If you think that the fallback for non-gzip content is not necessary, I will revert my change

Reply:
If API always returns gzip-encoded content, we need to revert this change.

Reply:
ok

Reply:
thanks

Reply:
Please do it

Reply:
I did here https://github.com/WOVNio/wovnrb/pull/130/commits/23a59b3b5b5d2628ff14a857f02a615035f20b04